### PR TITLE
(Feature) Adding id to user tooltip card

### PR DESCRIPTION
**Pull request by Lainow**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | !32593

image

### CODE REVIEW

File: src/DbUtils.php [hunk 1]
```diff
@@ -1821,6 +1821,7 @@ public function getUserName($ID, $link = 0, $disable_anon = false)
                     $user_params = [
                         'id'                 => $ID,
                         'user_name'          => $username,
+                        'login'              => $data['name'],
```
Comment (stonebuzz) [inline src/DbUtils.php:1824]: useless, ```login``` is already ahndle after (when we are not in a context of anonymization)
just the display part was missing

**Review by stonebuzz — CHANGES_REQUESTED** [review]

File: src/DbUtils.php [hunk 2]
```diff
@@ -1820,7 +1820,7 @@ public function getUserName($ID, $link = 0, $disable_anon = false)
 
                     $user_params = [
                         'id'                 => $ID,
-                        'user_name'          => $username,
+                        'user_name'          => $username
```
Comment (cedric-anne) [inline src/DbUtils.php:1823]: When your PR is finished, please, read the diff and remove changes that are useless. This kind of changes will pollute the "blame" view and the history of the file.

```suggestion
                        'user_name'          => $username,
```

**Review by cedric-anne — APPROVED** [review]

File: templates/components/user/info_card.html.twig [hunk 3]
```diff
@@ -52,8 +52,13 @@
                {{ user['user_name'] }}
             {% endif %}
          </h4>
-
          <div class="text-muted">
```
Comment (cedric-anne) [inline templates/components/user/info_card.html.twig:55]:
```suggestion

         <div class="text-muted">
```

File: templates/components/user/info_card.html.twig [hunk 4]
```diff
@@ -54,6 +54,12 @@
          </h4>
 
          <div class="text-muted">
+            {% if user['login'] is not empty %}
+                <div>
+                    <i class="fas fa-id-badge"></i>
```
Comment (orthagh) [inline templates/components/user/info_card.html.twig:59]:
```suggestion
                    <i class="fa-fw fas fa-id-badge"></i>
```

**Pull request merged**